### ISSUE TITLE
ssh cloning broken in version 4

### ISSUE THREAD

**Issue by richardscollin** [lambci/git-lambda-layer#11]
I just started using this for a side project of mine. I believe that the version 4 build ships a broken ssh that isn't linked properly to libfipscheck.so.1. This prevents ssh clones from working. Using the version 3 or 2 ARN link works fine.

run using `arn:aws:lambda:us-east-2:553035198032:layer:git:4` as a layer
```
+ ldd /opt/bin/ssh
	linux-vdso.so.1 =>  (0x00007ffc826fd000)
	libfipscheck.so.1 => not found
	libselinux.so.1 => /usr/lib64/libselinux.so.1 (0x00007fdcfddae000)
	libcrypto.so.10 => /var/lang/lib/libcrypto.so.10 (0x00007fdcfd950000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007fdcfd74c000)
	libldap-2.4.so.2 => /lib64/libldap-2.4.so.2 (0x00007fdcfd4fa000)
	liblber-2.4.so.2 => /lib64/liblber-2.4.so.2 (0x00007fdcfd2eb000)
	libutil.so.1 => /lib64/libutil.so.1 (0x00007fdcfd0e8000)
```

**Comment by mhart** [comment]
Oof, thanks for picking this up – a whole slew of changes were made to the Lambda OS recently that messed with the building of libraries.

I'll rebuild and check that it links correctly now

**Comment by mhart** [comment]
Should be fixed with `arn:aws:lambda:us-east-2:553035198032:layer:git:5` – let me know if that works for you, and thanks for picking this up!
